.travis.yml

```diff
@@ -5,25 +5,33 @@
 language: python
 
 python:
-  - 2.6
-  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
+  - 3.7-dev
```

Comment:
I think this could work (https://docs.travis-ci.com/user/languages/python/#development-releases-support).

I will update the branch now and let Travis give it a try.

Reply:
Had a look a few minutes after the build started, and alas looks like there's something weird with Python 3.8 in Travis. Two 3.8 builds immediately failed with

```bash
3.8-dev is not installed; attempting download
Downloading archive: https://s3.amazonaws.com/travis-python-archives/binaries/ubuntu/14.04/x86_64/python-3.8-dev.tar.bz2
$ curl -sSf -o python-3.8-dev.tar.bz2 ${archive_url}
curl: (22) The requested URL returned error: 403 Forbidden
Unable to download 3.8-dev archive. The archive may not exist. Please consider a different version.
```

I remember seeing something similar with Python 3.7 in Travis. It was a problem with the container image that the build was running on, and updating (but it was to download an `apt` package). Upgrading to the latest fixed. But I think it might be easier to leave 3.8 for later.

Reverted the commit.